index.js

```diff
@@ -320,7 +325,7 @@ function handleRequests() {
 /**
  * @type { typeof import('./index').init }
  */
-function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = 'api-docs') {
+function init(aApp, aPredefinedSpec = {}, aSpecOutputPath = undefined, aWriteInterval = 1000 * 10, aSwaggerUiServePath = 'api-docs', mongooseModels = []) {
```

Comment:
Could you also include this functionality in the `handleResponses` function? The `init` one is only for backwards-compatibility and basic usage only; everything else uses `handleResponses` and `handleRequests`.

Reply:
Done! I've also updated both `server_basic.js` and `server_advanced.js` examples.